deploy/ansible/playbooks/iroha-docker-cluster/iroha-deploy.yml

```diff
@@ -0,0 +1,16 @@
+  - hosts: locals
+    roles:
+      - { role: iroha-cluster-config-gen }
+
+  - hosts:
+    - iroha-east
+    - iroha-west
+    gather_facts: False
+    pre_tasks:
+    - name: install python 2
+      raw: test -e /usr/bin/python || (apt -y update && apt install -y python)
+      changed_when: False
+    roles:
+      - { role: docker, tags: docker }
+      - { role: filebeat }
+      - { role: iroha-cluster-deploy-node, tags: ["deliver", "deploy"] }
```

Comment:
Also, we need to create a role, which cleans up all docker containers by stopping them and removing the image:
docker stop $(docker ps -a -q)
docker rm $(docker ps -a -q)

Reply:
In case if we have made changes in docker image remotely, then another command should be executed docker rmi $(docker images -q) so please make a role for it also, so that we don't run it on each machine

Reply:
@neewy it is already implemented in a task - only iroha and postgres containers are stopped and removed before the playbook will run updated ones